packages/openbabel/test.R

```diff
@@ -0,0 +1,4 @@
+if (!requireNamespace("BiocManager", quietly = TRUE))
+  install.packages("BiocManager", repos = "http://cran.rstudio.com/")
+
+BiocManager::install("ChemmineOB")
```

Comment:
Please add a simple test that exercises the ChemmineOB package beyond installing it.